sympy/parsing/latex/lark/latex_parser.py

```diff
@@ -114,33 +114,33 @@ def relation(self, tokens):
             raise LaTeXParsingError() # TODO: Fill descriptive error message.
 
     def add(self, tokens):
-        return sympy.Add(tokens[0], tokens[2], evaluate=False)
+        return sympy.Add(tokens[0], tokens[2])
 
     def sub(self, tokens):
         if len(tokens) == 2:
             return -tokens[1]
         elif len(tokens) == 3:
-            return sympy.Add(tokens[0], -tokens[2], evaluate=False)
+            return sympy.Add(tokens[0], -tokens[2])
 
     def mul(self, tokens):
         if len(tokens) == 2:
-            return sympy.Mul(tokens[0], tokens[1], evaluate=False)
+            return sympy.Mul(tokens[0], tokens[1])
         elif len(tokens) == 3:
-            return sympy.Mul(tokens[0], tokens[2], evaluate=False)
+            return sympy.Mul(tokens[0], tokens[2])
         else:
             raise LaTeXParsingError() # TODO: fill out descriptive error message
 
     def div(self, tokens):
-        return sympy.Mul(tokens[0], sympy.Pow(tokens[2], -1, evaluate=False), evaluate=False)
+        return sympy.Mul(tokens[0], sympy.Pow(tokens[2], -1))
 
     def superscript(self, tokens):
-        return sympy.Pow(tokens[0], tokens[2], evaluate=False)
+        return sympy.Pow(tokens[0], tokens[2])
 
     def fraction(self, tokens):
-        return sympy.Mul(tokens[1], sympy.Pow(tokens[2], -1, evaluate=False), evaluate=False)
+        return sympy.Mul(tokens[1], sympy.Pow(tokens[2], -1))
```

Comment:
After removing `evaluate` option, it is now possible to build sympy expressions without `Add, Mul, Pow`. I'd prefer to write division as `tokens[1] / tokens[2]`

Reply:
How would this change interact with the `evaluate(False)` context manager? I.e., would
```python
with evaluate(False):
    parse_latex_lark(r"\frac12")
```
still work as expected?

Reply:
It looks like it works as expected:
```python
In [14]: with evaluate(False):
    ...:    expr = parse_latex_lark(r"\frac12")
In [15]: expr
Out[15]: 1/2
In [16]: srepr(expr)
Out[16]: 'Mul(Integer(1), Pow(Integer(2), Integer(-1)))'
```

Reply:
Using `evaluate(False)` context manager means that `evaluate=False` is passed throughout any depth inside the block,
without drilling down `evaluate=False`, so it works.

Reply:
I've made the requested change, so I'm marking this conversation as resolved.